lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -222,6 +225,28 @@ describe('GatherRunner', function() {
     });
   });
 
+  it('tells the driver to block given URL patterns when blockedUrlPatterns is given', () => {
+    const receivedUrlPatterns = [];
+    const urlPatterns = ['http://*.evil.com', '.jpg', '.woff2'];
+    const driver = getMockedEmulationDriver(null, null, null, params => {
+      receivedUrlPatterns.push(params.url);
+    });
+
+    return GatherRunner.setupDriver(driver, {flags: {blockedUrlPatterns: urlPatterns.slice()}})
+      .then(() => assert.equal(receivedUrlPatterns.sort().join(), urlPatterns.sort().join()));
```

Comment:
not sure if `assert.equal` means much on these arrays. do we need `deepEqual` or `deepStrictEqual` ? /me shrugs.